### PR TITLE
Code review:Ruby-Enumerables

### PR DESCRIPTION
**Pull request by bafiam**
#### What does this PR do?
1. Consist of the test cases to test the Ruby-Enumerables

#### How should this be manually tested?
Clone the repo then navigate to the project directory, on the terminal type `rspec

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
## Hi team!
### Nice job! Project looks! :+1: 

### But your project needs a few changes before it can be approved.


#### General

*  Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history 
screenshot from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
*  Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. screenshot from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)

* Specs for each method should be wrapped in one describe block. As in example here https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples .  (screenshot  from  Pathwright)
#### [REQUIRED] 
* Specs for each method should be wrapped in one `describe` block. As in example here https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples . 
screenshot from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)


*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding *

File: spec/m_enumerables_spec.rb [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  let(:arr_fruits) { %w[apple orange banana bacovo guava basa] }
+  describe 'Given either block or array' do
+    describe '#my_each' do
+      it 'return enumerable when no block given' do
+        expect([1, 2, 3, 4, 5].my_each).to be_an(Enumerator)
+      end
+
+      it 'return same data with ruby each' do
+        output_my = [1, 2, 3, 4, 5].my_each { |num| }
+        output_each = [1, 2, 3, 4, 5].each { |num| }
```
Comment (Forison) [inline spec/m_enumerables_spec.rb]: **[optional]**
You don't have to perform``` output_each = [1, 2, 3, 4, 5].each { |num| }``` , you simply expect ```[1,2,3,4,5]```

File: spec/m_enumerables_spec.rb [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        expect([1, 2, 4, 2].my_count(2)).to be 2
+      end
+      it 'Will return number of items true to a block, if block is given' do
+        expect([1, 2, 4, 2].my_count { |x| (x % 2).zero? }).to be 3
+      end
+    end
+  end
+  describe '#my_map' do
+    context 'Returns a new array with the results of running block once for every element in enum' do
+      it 'should return an array with blocks result ' do
```
Comment (Forison) [inline spec/m_enumerables_spec.rb]: * [ ] ``` my_map``` returns enum when no block is given, that is also functionality that ought to be tested.

File: spec/m_enumerables_spec.rb [hunk 3]
```diff
@@ -0,0 +1,175 @@
+# frozen_string_literal: true
+
+require 'rspec'
+require_relative '../m_enumerables.rb'
+
+describe 'my_enumerables' do
```
Comment (Forison) [inline spec/m_enumerables_spec.rb]: **[optional]**
* [ ] we could also use this ```RSpec.describe Enumerable do``` without line 3🙂

**Pull request merged**